docs/folia/admin/reference/faq.md

```diff
@@ -1,6 +1,7 @@
 ---
 slug: /faq
 title: FAQ
+description: Some questions asked frequently by our community.
```

Comment:
```suggestion
description: Questions frequently asked by our community, answered by us!
```
or
```suggestion
description: Frequently asked Folia questions.
```
to match [this](https://github.com/PaperMC/docs/pull/244/files#diff-3a90be20010c1a760f49bffbcb5d31411355d21d6f8d22ee64138e440547639bR3) or [this](https://github.com/PaperMC/docs/pull/244/files#diff-674fd41868c4a4528a24b68603cf419cabe513062210ff64576f0eb614ea570aR3)